solutions/count_vowels.py

```diff
@@ -0,0 +1,29 @@
+def count_vowels(input_string):
+    """
+    Counts the number of vowels in a given string.
+
+    Args:
+        input_string (str): The string to analyze for vowels.
+
+    Returns:
+        int: The total number of vowels (a, e, i, o, u) in the string,
+             including both uppercase and lowercase vowels.
+
+    Example:
+        >>> count_vowels("Hello, World!")
+        3
+        >>> count_vowels("Python")
+        1
+    """
```

Comment:
Add assertion to make sure the input value is of a string type.
```suggestion
    """
    assert isinstance(input_string, str), "The input value should be a string"
```